TerminalDocs/customize-settings/key-bindings.md

```diff
@@ -290,6 +290,27 @@ This opens a specific tab depending on the index.
 | ---- | --------- | ------- | ----------- |
 | `index` | Required | Integer | Tab that will open based on its position in the tab bar (starting at 0). |
 
+### Rename the Tab
```

Comment:
This may need to be badged with the Preview tag for Preview 1.2 (when it comes out). Not sure how we should track 1.2 things when 1.1 is actively in preview. @cinnamon-msft?

Reply:
Ah yeah. We should make a release-1.2 branch and have this PR point to there instead of master. Then, the Preview tag can be added and it will all go live once we merge the release-1.2 branch into the master and live branches.
My apologies for not creating this branch sooner!

Reply:
Okay, just made the release-1.2 branch 😄

Reply:
Okay made the PR point to the new branch! Not sure what badging with the preview tag means though, is that something I'd have to do?